meetings/2022-09-08.md

```diff
@@ -44,51 +46,72 @@
-
->ACTION: RG to create issue to note the cases in which client may want to update its subscriptions or connections.
-
-* SC: Before discussing that issue, does https://github.com/solid/notifications/issues/97 need a follow-up? Does 103 supersede 97? Close 97?
+* SC: Notification Channel Resource is the current term for the resource that includes some information about the notification channel/service and its capabilities.
+* SC: Some people find `http://www.w3.org/ns/solid#storageDescription` useful - at least on the Solid Protocol level. Current ED explains how to take advantage of it.. in addition to `describedby`. Thoughts?
+* RG: Is it two different mechanisms available all the time?
+* SC: Same same different. resource-specific or storage-wide cases.
+* RG: But in the end it links to two places to get that information.
+* AC: Asking for clarity: in the link header, is all of the information in the header available or ?
+* RG: No. Will the info be in describedby or the storageDescripiton. How would they know about the storage description when they access the first resource.
+* SC: examples:
+
+```http
+HEAD /foo
+
+Link: </notification-resource>; rel="describedby"
+Link: </notification-resource>; rel="http://www.w3.org/ns/solid#storageDescription"
+```
+
+* SC: None, one, or both of those headers may appear.
+* RG: That clarifies.
+* RG: Can we have a non-normative section describing guidance or that they can even split this information.
+* SC: Perhaps. Right now just with terminology around the intention i.e., resource-specific or storage-wide.
+* SC: https://solid.github.io/notifications/protocol#server-link-resource-notification-channel :
+>When a server wants to enable applications to discover Notification Channels available to a **given resource**, the server MUST advertise the associated resources describing the channel information by responding to an HTTP request including a Link header with the rel value of describedby [POWDER-DR] and the Description Resource as link target [RFC8288].
+* SC: https://solid.github.io/notifications/protocol#server-link-storage-notification-channel :
+>When a server wants to enable applications to discover Notification Channels available to a **storage** in which a given resource is in, the server MUST advertise the associated resources describing the channel information by responding to an HTTP request including a Link header with the rel value of http://www.w3.org/ns/solid#storageDescription [SOLID-PROTOCOL] and the Description Resource as link target [RFC8288].
+* AC: For the sake of interoperability, I the notifications protocol, if we required describedby, and effectively said, the server could expose all the other headers but then maybe we don't need to say that. The two different mechanisms would be available if used specific against Solid Protocol server. But describedby would me more widely and generally understandable.
```

Comment:
The blank lines are necessary to end the MD quotations, and the 2-space indents are required to "embed" the quotations in the SC bullets
```suggestion
  >When a server wants to enable applications to discover Notification Channels available to a **given resource**, the server MUST advertise the associated resources describing the channel information by responding to an HTTP request including a Link header with the rel value of describedby [POWDER-DR] and the Description Resource as link target [RFC8288].

* SC: https://solid.github.io/notifications/protocol#server-link-storage-notification-channel :
  >When a server wants to enable applications to discover Notification Channels available to a **storage** in which a given resource is stored, the server MUST advertise the associated resources describing the channel information by responding to an HTTP request including a `Link` header with the `rel` value of `http://www.w3.org/ns/solid#storageDescription` [SOLID-PROTOCOL] and the Description Resource as link target [RFC8288].

* AC: For the sake of interoperability, I the notifications protocol, if we required describedby, and effectively said, the server could expose all the other headers but then maybe we don't need to say that. The two different mechanisms would be available if used specific against Solid Protocol server. But describedby would me more widely and generally understandable.
```